UI/src/components/features/RipaPageContainer.vue

```diff
@@ -43,6 +47,7 @@ export default {
       'isAdmin',
       'invalidUser',
       'isOnlineAndAuthenticated',
+      'isAuthenticated',
```

Comment:
We probably should use the existing isOnlineAndAuthenticated property since we don't want to check if the user is offline.